packages/mobile/src/home/reducers.ts

```diff
@@ -18,6 +18,7 @@ export interface Notification {
   maxVersion?: string
   countries?: string[]
   blockedCountries?: string[]
+  external?: boolean
```

Comment:
Let's rename to keep it consistent with the naming used for push notification: https://github.com/celo-org/wallet/blob/c2cf2292a01d11fe53f23dae91f9a463b46da747/packages/mobile/src/firebase/notifications.ts#L96
```suggestion
  openExternal?: boolean
```